app/models/category.rb

```diff
@@ -1,7 +1,33 @@
+# Encoding: utf-8
 class Category < ApplicationRecord
   has_many :issues
   scope :ordered_by_name, -> { order(:name) }
 
+  REGEXP_NAME = /\A[[:alnum:] ,-]*\z/
+  REGEXP_TAGS = /\A[[:alnum:] _]*\z/
+
+  validates :name, :description, :tags,
+            presence: true
+  validates :name, uniqueness: { case_sensitive: false },
+                   length: { minimum: 5, maximum: 100 },
+                   format: { with: REGEXP_NAME,
+                             message: 'Назва може містити тільки
+                                       літери, цифри, коми і дефіс.' }
+  validates :description, length: { minimum: 20, maximum: 300 }
+  validates :tags, length: { minimum: 2, maximum: 100 },
+                   format: { with: REGEXP_TAGS,
+                             message: 'Теги можуть містити тільки
+                                       літери, цифри і знак _.
+                                       Розділяти теги потрібно пробілом.' }
+
+  def name=(s)
```

Comment:
why you doing this capitalization? just save as is. Admin will create categories with correct name. This is not needed code which can cause bugs.

Rule: less code = less bugs